moveit_ros/moveit_servo/src/collision_check.cpp

```diff
@@ -53,27 +53,27 @@ namespace moveit_servo
 CollisionCheck::CollisionCheck(rclcpp::Node::SharedPtr node, const ServoParametersPtr& parameters,
                                const planning_scene_monitor::PlanningSceneMonitorPtr& planning_scene_monitor)
   : node_(node)
-  , parameters_(parameters)
+  , servo_parameters_(parameters)
```

Comment:
Not sure if this renaming is really worth it, it touches a lot of lines and will definitely make syncing difficult in the future.

Reply:
yeah, i tend to agree

Reply:
Ok, I'll remove the rename.  I thought it would make it clearer because there is a pose tracker parameters struct also.  I agree that this is unrelated to this PR and shouldn't be here.